debian/rules

```diff
@@ -30,16 +30,16 @@ endif
 	dh $@ --with python3,bash-completion,systemd --buildsystem=pybuild
 
 override_dh_auto_build:
+	# fail early if version.py wasn't updated to match d/changelog's version
+	# only take into account the actual upstream version, discarding ubuntu and
+	# backport suffixes
+	python3 tools/check-versions-are-consistent.py
 	dh_auto_build
 	make -C apt-hook build
 	make -C debian/po build
 
 override_dh_auto_test:
 ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
```

Comment:
Not quite sure but do we need the `ifeq` if we no conditional run inside?

Reply:
there is a filter for the `nocheck` option. If this is present, there is no testing when building at all

Reply:
Yes, and the lack of indentation makes it hard to read but there is code that only runs inside this `ifeq`